src/main/java/seedu/awe/model/ModelManager.java

```diff
@@ -226,6 +227,27 @@ public void setTransactionSummary(HashMap<Person, Cost> summary) {
         return payments;
     }
 
+    @Override
+    public void deletePersonFromGroups(Person person) {
+        List<Group> groupList = new ArrayList<>();
+        List<Group> groups = addressBook.getGroupList();
+        for (int i = 0; i < groups.size(); i++) {
+            if (groups.get(i).isPartOfGroup(person)) {
```

Comment:
With the chaining of method calls (which decreases readability) and the fact that this code block seems to follow an arrowhead format, perhaps it might be a good idea to refactor this part into a method on its own? Doesnt have to be a method that overrides from the interface.

Reply:
Noted, have made the appropriate changes.